### PR TITLE
Selective metadata packages list

### PR DESCRIPTION
**Pull request by fiskus**
# Description

Spec: https://github.com/quiltdata/specs/pull/44
Docs: https://github.com/quiltdata/quilt/pull/2942

# TODO

- [ ] Unit tests
- [ ] ~Automated tests (e.g. Preflight)~
- [x] Documentation
    - [ ] ~[Python: Run `build.py`](../gendocs/build.py) for new docstrings~
    - [x] JavaScript: basic explanation and screenshot of new features
- [ ] [Changelog](CHANGELOG.md) entry

### CODE REVIEW

**Comment by fiskus** [comment]
@nl0 can you review it today?

File: catalog/app/containers/Bucket/PackageList/PackageList.tsx [hunk 1]
```diff
@@ -136,10 +138,123 @@ function Counts({ counts, total }: CountsProps) {
   )
 }
 
-const usePackageStyles = M.makeStyles((t) => ({
-  root: {
+const useRevisionAttributes = M.makeStyles((t) => ({
```
Comment (nl0) [inline catalog/app/containers/Bucket/PackageList/PackageList.tsx:141]:
```suggestion
const useRevisionAttributesStyles = M.makeStyles((t) => ({
```

File: catalog/app/containers/Bucket/PackageList/PackageList.tsx [hunk 2]
```diff
@@ -136,10 +138,123 @@ function Counts({ counts, total }: CountsProps) {
...
+}
+
+function RevisionMeta({ sections }: RevisionMetaProps) {
+  const classes = useRevisionMetaStyles()
+
+  return (
+    <div className={classes.root}>
+      {sections.map((section, i) => (
+        <div className={classes.section} key={`${i}+${section}`}>
+          {typeof section === 'string' && section}
```
Comment (nl0) [inline catalog/app/containers/Bucket/PackageList/PackageList.tsx:218]: i'd use ternary here, but whatever

**Review by nl0 — APPROVED** [review]
looks good, tho im little worried that package list and package revision list views are diverging -- same data pieces are shown in different places in these views. maybe we should converge the layout somewhat? at least maybe show the message in the same place idk 🤷 

Screenshot 2022-07-21 at 13 43 58
Screenshot 2022-07-21 at 13 44 09

**Comment by fiskus** [comment]
Let's converge the layout in the next iterations (I'll continue to work on this feature, introducing search in particular). But I also want that they preserve visually different so user be able to comprehend what page he sees in milliseconds :)

**Pull request merged**